core/constant.ts

```diff
@@ -0,0 +1,9 @@
+import { config } from 'dotenv';
+import { User } from './objects/User';
+config();
```

Comment:
config() call must be placed as early in your app as possible, that's why placing it in playwright.config file would be better. You even have a commented line there already - // require('dotenv').config();